fml/paths.cc

```diff
@@ -29,5 +29,25 @@ std::string JoinPaths(std::initializer_list<std::string> components) {
   return stream.str();
 }
 
+std::string SanitizeURIEscapedCharacters(const std::string& str) {
```

Comment:
I feel that this function is a nice candidate for a unit test which both helps ensure correctness, and helps the reader understand what the function is doing. Do we have unit tests for fml yet?

Reply:
done